packages/backend/src/queryBuilder.ts

```diff
@@ -108,23 +108,27 @@ export const replaceUserAttributes = (
 
     return sqlAttributes.reduce<string>((acc, sqlAttribute) => {
         const attribute = sqlAttribute.replace(userAttributeRegex, '$1');
-        const userValue: string | null | undefined =
-            userAttributes[attribute]?.[0]; // Pick first value, where user value takes precedence over group values
+        const attributeValues: string[] | undefined = userAttributes[attribute];
 
-        if (userValue === undefined) {
+        if (attributeValues === undefined) {
             throw new ForbiddenError(
                 `Missing user attribute "${attribute}" on ${filter}: "${sqlFilter}"`,
             );
         }
-        if (userValue === null) {
+        if (attributeValues.length === 0) {
             throw new ForbiddenError(
                 `Invalid or missing user attribute "${attribute}" on ${filter}: "${sqlFilter}"`,
             );
         }
 
         return acc.replace(
             sqlAttribute,
-            `${stringQuoteChar}${userValue}${stringQuoteChar}`,
+            attributeValues
+                .map(
+                    (attributeValue) =>
+                        `${stringQuoteChar}${attributeValue}${stringQuoteChar}`,
+                )
+                .join(','),
```

Comment:
would it look cleaner to have a space after the commas?

Reply:
The attribute can only be set by admins so not a big concern now but we should be sql escaping this value, since it's user provided